packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -185,8 +185,25 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
             data-command-key={ index }
             aria-haspopup={ hasSubmenuItems(item) }
             >
-            { (hasIcon) ? this._renderIcon(item) : (null) }
-            <span className='ms-CommandBarItem-commandText ms-font-m ms-font-weight-regular' aria-hidden='true' role='presentation'>{ item.name }</span>
+            { (() => {
+              if (item.href) {
```

Comment:
why use `() => {}` ?. Also would be better if we could have a single return and replace if-else with `? :`.